src/models/shape_loader.h

```diff
@@ -1,22 +1,48 @@
-#ifndef WIRE_VOLUME_MODEL_DB_LOADER_H_
-#define WIRE_VOLUME_MODEL_DB_LOADER_H_
+#ifndef ED_MODELS_SHAPE_LOADER_H_
+#define ED_MODELS_SHAPE_LOADER_H_
 
 #include <geolib/datatypes.h>
 #include <tue/config/reader.h>
 
+#include <string>
+#include <vector>
+#include <sstream>
+
 namespace ed
 {
 
 namespace models
 {
 
+/**
+ * @brief The ModelOrFile enum This is used to determine the URI type in SDF.
+ */
+enum ModelOrFile
+{
+    MODEL,
+    FILE
+};
+
+std::vector<std::string> split(std::string& strToSplit, char delimeter);
```

Comment:
Use on paths?
Have a `const` for `strToSplit`?

Reply:
Done